src/Common/tests/System/Net/Prerequisites/Servers/CoreFxNetCloudService/WebServer/WebSocket/EchoWebSocket.ashx.cs

```diff
@@ -15,12 +15,15 @@ namespace WebServer
     public class EchoWebSocket : IHttpHandler
     {
         private const int MaxBufferSize = 128 * 1024;
+        private bool _replyWithPartialMessages;
```

Comment:
You should initialize this to 'false' so that the same behavior without this querystring parameter happens, i.e. reply with full messages.